services/src/main/java/org/apache/unomi/services/impl/rules/RulesServiceImpl.java

```diff
@@ -96,6 +100,10 @@ public void setRulesStatisticsRefreshInterval(Integer rulesStatisticsRefreshInte
         this.rulesStatisticsRefreshInterval = rulesStatisticsRefreshInterval;
     }
 
+    public void setOptimizedRulesActivated(Boolean optimizedRulesActivated) {
+        this.optimizedRulesActivated = optimizedRulesActivated;
+    }
+
```

Comment:
This seem's not used anymore.

Reply:
Yes it is, it is used by Blueprint configuration.